src/main/scala/com/fulcrumgenomics/fgsv/util/PutativeBreakpoint.scala

```diff
@@ -5,52 +5,69 @@ package com.fulcrumgenomics.fgsv.util
- rightPos: Int,
- sameStrand: Boolean,
- evidence: EvidenceType
-) extends Ordered[PutativeBreakpoint] {
+case class PutativeBreakpoint(id: Long = 0,
+                              leftRefIndex: Int,
+                              leftPos: Int,
+                              leftPositive: Boolean,
+                              rightRefIndex: Int,
+                              rightPos: Int,
+                              rightPositive: Boolean,
+                              evidence: EvidenceType
+                             ) extends Ordered[PutativeBreakpoint] {
   /** Returns true the other breakpoint has the same break ends.  This compares all fields except the evidence type. */
   def sameBreakEnds(that: PutativeBreakpoint): Boolean = {
     this.leftRefIndex == that.leftRefIndex &&
       this.leftPos == that.leftPos &&
+      this.leftPositive == that.leftPositive &&
       this.rightRefIndex == that.rightRefIndex &&
       this.rightPos == that.rightPos &&
-      this.sameStrand == that.sameStrand
+      this.rightPositive == that.rightPositive
   }
 
   /** Defines an ordering over breakpoints, first by genomic coordinates of the left then right end, then evidence type. */
   def compare(that: PutativeBreakpoint): Int = {
     var result: Int = 0
     if (result == 0) result = this.leftRefIndex - that.leftRefIndex
     if (result == 0) result = this.leftPos - that.leftPos
+    if (result == 0) result = this.leftPositive.compare(that.leftPositive)
     if (result == 0) result = this.rightRefIndex - that.rightRefIndex
     if (result == 0) result = this.rightPos - that.rightPos
+    if (result == 0) result = this.rightPositive.compare(that.rightPositive)
     if (result == 0) result = EvidenceType.indexOf(this.evidence) - EvidenceType.indexOf(that.evidence)
     result
   }
+
+  @Deprecated
+  /** TODO: remove this one other code is fixed. */
+  def sameStrand: Boolean = this.leftPositive == this.rightPositive
 }
 
 object PutativeBreakpoint {
   /** Builds a breakpoint from two alignment blocks, with the left-side having by lower genomic coordinate. */
   def apply(b1: AlignmentBlock, b2: AlignmentBlock, evidence: EvidenceType): PutativeBreakpoint = {
-    val (first, second) = if (b1.range <= b2.range) (b1, b2) else (b2, b1)
-    PutativeBreakpoint(
-      leftRefIndex  = first.range.refIndex,
-      leftPos       = first.range.end,
-      rightRefIndex = second.range.refIndex,
-      rightPos      = second.range.start,
-      sameStrand    = first.positiveStrand == second.positiveStrand,
+    val x = new PutativeBreakpoint(
+      leftRefIndex  = b1.range.refIndex,
+      leftPos       = if (b1.positiveStrand) b1.range.end else b1.range.start,
+      leftPositive  = b1.positiveStrand,
+      rightRefIndex = b2.range.refIndex,
+      rightPos      = if (b2.positiveStrand) b2.range.start else b2.range.end,
+      rightPositive = b2.positiveStrand,
       evidence      = evidence
     )
+
+    if (b1.range < b2.range) x else x.copy(
```

Comment:
This isn't the most efficient way to do this, but it's the easiest to think about so for now I'm doing it.  Once I have enough tests to be confident it's all right, we can refactor to not create two breakpoints.